src/mmrelay/plugins/mesh_relay_plugin.py

```
@@ -159,25 +162,22 @@ def matches(self, event):
             return bool(match)
         return False
 
-    async def handle_room_message(self, room, event, full_message):
+    async def handle_room_message(self, room, event, full_message) -> bool:
```

Ruff (0.14.6)

Unused method argument: `full_message`

(ARG002)